v2/paths/recognition/FacesSearch.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
```

Comment:
`Face` has been renamed to `Image` so here the file would be named `ImagesSearch`.

Reply:
Resolved